backend/app.js

```diff
@@ -1,28 +1,32 @@
-var createError = require('http-errors');
-var express = require('express');
-var path = require('path');
-var cookieParser = require('cookie-parser');
-var logger = require('morgan');
-
-var indexRouter = require('./routes/index');
-var usersRouter = require('./routes/users');
-var testRouter = require('./routes/testAPI');
+var createError = require("http-errors");
+var express = require("express");
+var path = require("path");
+var cookieParser = require("cookie-parser");
+var logger = require("morgan");
+if (process.env.NODE_ENV !== "production") {
+  require("dotenv").config();
+  console.log(process.env);
```

Comment:
Don't print environment variables because it will print our sensitive info in plain text!